### PR TITLE
r-gui 1.66 (updated formula)

### PR DESCRIPTION
**Pull request by joshbode**
- new version (1.65 -> 1.66)

### CODE REVIEW

File: r-gui.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class RGui < Formula
 
   homepage "http://cran.r-project.org/bin/macosx/"
-  url "http://cran.r-project.org/bin/macosx/Mac-GUI-1.65.tar.gz"
-  sha1 "9c6aaabbfe2307f4cedd6a4b49ffda88952a8a1e"
+  url "http://cran.r-project.org/bin/macosx/Mac-GUI-1.66.tar.gz"
+  sha1 "82f4ac0b2b2c7ec2f017e8c61b9b09dbd9887264"
```
Comment (dpo) [inline r-gui.rb]: We now use the `sha256`. You can find it out with `shasum -a 256`.

Comment (dpo) [inline r-gui.rb]: It would also be great if you could add a `desc` field with a brief description of the software.

**Comment by joshbode** [comment]
Will do - just noticed the build error. Thanks

**Comment by dpo** [comment]
Thanks! Please squash your commits.

**Comment by joshbode** [comment]
I hope this worked - I thought I had squished my recent commits, but I've got no idea how to squash after merging a year of commits from upstream into my old r-gui branch. So I killed the branch on my fork and branched from origin/master anew.

Moral of story - follow the instructions (!) and start from a fresh branch.

**Comment by dpo** [comment]
Looks good. Thanks!

**Comment by joshbode** [comment]
No problem! Thank you for maintaining the Tap, home to some of my favourite Formulae ;)

**Pull request closed without merging**